parca-agent.bpf.c

```diff
@@ -23,6 +23,10 @@
 #include <bpf_helpers.h>
 #include <bpf_tracing.h>
 
+volatile const char bpf_metadata_name[] SEC(".rodata") =
+    "parca-agent (https://github.com/parca-dev/parca-agent)";
+volatile const char bpf_metadata_version[] SEC(".rodata") = "v0.7.0";
```

Comment:
I'm sure you're already on it, better to inject this dynamically, either while building or loading the BPF program.

Reply:
Adding this without a version sounds good to me. Let's try to eliminate any sort of possible confusions.